pnpm-workspace.yaml

```
@@ -0,0 +1,4 @@
+packages:
+  - packages/downloader
+  - packages/server
+  - packages/ui
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)